backend/src/app/db.clj

```diff
@@ -59,14 +60,14 @@
 
 (defmethod ig/init-key ::pool
   [_ {:keys [migrations metrics] :as cfg}]
-  (log/infof "initialize connection pool '%s' with uri '%s'" (:name cfg) (:uri cfg))
+  (log/infof "initialize connection pool '%s' with uri '%s'" (name (:name cfg)) (:uri cfg))
   (instrument-jdbc! (:registry metrics))
   (let [pool (create-pool cfg)]
     (when (seq migrations)
       (with-open [conn ^AutoCloseable (open pool)]
         (mg/setup! conn)
-        (doseq [[mname steps] migrations]
-          (mg/migrate! conn {:name (name mname) :steps steps}))))
+        (doseq [[name steps] migrations]
+          (mg/migrate! conn {:name (c/name name) :steps steps}))))
```

Comment:
you can use the `name` function in `app.common.data` it's safer than the clojure default

Reply:
:+1: